daffodil-test/src/test/scala/org/apache/daffodil/section23/dfdl_expressions/TestBitFunctions.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.section23.dfdl_expressions
+
+import org.apache.daffodil.tdml.Runner
+import org.junit._
+
+
+object TestBitFunctions {
+  val testDir2 = "/org/apache/daffodil/section23/dfdl_functions/"
+
+  val runner2 = Runner(testDir2, "BitFunctions.tdml")
+//  val runner2_utf8 = Runner(testDir2, "Functions_UTF8.tdml")
```

Comment:
remove commented out code lines in this file

Also suggest rename testDir2 to just testDir, and runner2 to just runner.